olp-cpp-sdk-core/include/olp/core/utils/WarningWorkarounds.h

```diff
@@ -21,23 +21,25 @@
 
 /**
  * @file
- * Contains utilities to work around compiler warnings.
+ * @brief Contains utilities used to work around compiler warnings.
```

Comment:
I am not sure if @brief is accepted here, see http://cs.swan.ac.uk/~csoliver/ok-sat-library/internet_html/doc/doc/Doxygen/1.7.6.1/html/commands.html#cmdfile

Reply:
It is. I tested it, and without "@brief" the description is not added at the beginning of the Doxygen page.